libvast/src/system/exporter.cpp

```diff
@@ -135,6 +135,7 @@ behavior exporter(stateful_actor<exporter_state>* self, expression expr,
   self->set_exit_handler(
     [=](const exit_msg& msg) {
       VAST_DEBUG(self, "received exit from", msg.source, "with reason:", msg.reason);
+      self->send<message_priority::high>(self->state.index, self->state.id, 0);
```

Comment:
The handler on the INDEX currently has this form:
```
    [=](const uuid& query_id, size_t num_partitions) {
```

In a separate commit (or PR if you wish), how about using fixed-size integers for more robustness?

Reply:
> I don't understand.

The problem is that `size_t` can have different width. Say a client sends a message from a 32-bit platform but the server runs on 64-bit architecture. Then the `size_t` handler wouldn't fire. 

>  Do you want the type of num_partitions changed? To `uint64_t`?

Yeah, 32 bits might suffice though.